community/git.py

```diff
@@ -179,3 +179,23 @@ def get_upstream_deploy_url():
     deploy_url = 'https://%s.github.io/%s' % (owner, path)
 
     return deploy_url
+
+
+def get_external_url(org_name, repo_name, file_name):
+    """
+    Obtain external url
+    """
+    # Use environment variable to switch between sources
+    external_site = os.environ.get('EXTERNAL_SITE')
+    if external_site is None:
+        # use GitHub as default
+        url = ('http://' + org_name + '.github.io/'
```

Comment:
This logic is `get_upstream_deploy_url()` above.

And if you use that, IIRC, you should not need `EXTERNAL_SITE` because it lets forks have their own website, and we can enhance it if it isnt working for you.

But Netlify support could be added, now.

GitLab support should be done when the rest of the module supports GitLab as a remote. (very separate issue)

And if you add arg `path` to `get_upstream_deploy_url()` , it is probably exactly what you need.

Reply:
For this PR, Netlify is not needed. I can remove `get_external_url`, reuse and add `owner`, `path` options to `get_upstream_deploy_url`. One problem I am wondering is that would break the semantic meanging of `upstream_deploy_url`.